### PR TITLE
TF frontend: add rint op

### PR DESCRIPTION
**Pull request by alter-xp**
In order to enrich the support of TF operator and the need of practical work, we add rint op in tensorflow frontend.

@giuseros

### CODE REVIEW

File: python/tvm/relay/frontend/tensorflow.py [hunk 1]
```diff
@@ -2391,6 +2391,7 @@ def _impl(inputs, attr, params, mod):
     "ReverseV2": _reverse_v2(),
     "RightShift": AttrCvt("right_shift"),
     "Round": AttrCvt("round"),
+    "Rint": AttrCvt("round"),
```
Comment (siju-samuel) [inline python/tvm/relay/frontend/tensorflow.py:2394]: preserve the alphabetical order. keep Rint above Round.

File: tests/python/frontend/tensorflow/test_forward.py [hunk 2]
```diff
@@ -3520,6 +3520,23 @@ def _test_forward_softsign(shape):
     _test_forward_softsign([2, 5, 2, 5])
 
 
+def test_forward_rint():
+    """test operator rint """
+
+    def _test_forward_rint(shape):
+        tf.disable_eager_execution()
+        np_data = np.random.uniform(1, 100, size=shape).astype(np.float32)
```
Comment (siju-samuel) [inline tests/python/frontend/tensorflow/test_forward.py:3528]: include -ve floats also for testing.

**Comment by giuseros** [comment]
Hi @alter-xp ,
Given you address @siju-samuel comments, this LGTM

File: tests/python/frontend/tensorflow/test_forward.py (hunk 2 above)
Comment (alter-xp) [reply tests/python/frontend/tensorflow/test_forward.py:3528]: Thank you for your comments, but I don't get your point.
What does '- ve floats' mean?

File: python/tvm/relay/frontend/tensorflow.py (hunk 1 above)
Comment (alter-xp) [reply python/tvm/relay/frontend/tensorflow.py:2394]: 🆗

File: tests/python/frontend/tensorflow/test_forward.py (hunk 2 above)
Comment (siju-samuel) [reply tests/python/frontend/tensorflow/test_forward.py:3528]: use 
`np_data = np.random.uniform(-100, 100, size=shape).astype(np.float32)`

Comment (alter-xp) [reply tests/python/frontend/tensorflow/test_forward.py]: thinks!

Comment (alter-xp) [reply tests/python/frontend/tensorflow/test_forward.py]: thanks!

**Review by siju-samuel — APPROVED** [review]
LGTM

**Comment by siju-samuel** [comment]
Thanks @alter-xp @giuseros . This PR is merged.

**Pull request merged**